pylint/checkers/exceptions.py

```diff
@@ -333,19 +334,39 @@ def _check_raise_missing_from(self, node: nodes.Raise) -> None:
         # like `exc.with_traceback(whatever)`. We won't analyze these, we'll just assume
         # there's a violation on two simple cases: `raise SomeException(whatever)` and `raise
         # SomeException`.
+
+        need_to_raise = False
```

Comment:
I would be okay with two `add_message` instead of this variable. Would also save some assignments as we can just pass `""` or the actual `except_as_exc`.